tests/sentry/integrations/github/test_integration.py

```diff
@@ -114,6 +116,9 @@ def _stub_github(self):
                 "full_name": "Test-Organization/baz",
                 "default_branch": "master",
             },
+            "archived": {
+                "archived": True,
```

Comment:
Somewhere in the get_repositories we discard repos that are archived.
This is simply to test that part of the code.